packages/rich-text-editor/src/RichTextEditor/components/ToggleIconButton/ToggleIconButton.scss

```diff
@@ -33,76 +33,120 @@
     right: calc(-1 * #{$focus-ring-offset});
     bottom: calc(-1 * #{$focus-ring-offset});
   }
+
+  &[aria-disabled="true"] {
```

Comment:
We should probably use a class here. Even though I don't think this property will change, there's an explicit relationship between classes and styling which we don't get from this.

Usually this is an issue when people use an element type as a selector, but the principle stands that if someone changes this property, they wouldn't expect that they would also be changing something related to styling.

Reply:
that's fair, happy to change that over :)